packages/docs/docs/contributing/troubleshooting.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Troubleshooting
+---
+
+This guide helps you resolve common issues when developing for Actual.
+
+## Type Errors
+
+### Issue: TypeScript compilation errors
+
+**Solution:**
+
+1. Run `yarn typecheck` to see all type errors
+2. Check if types are imported correctly
+3. Look for existing type definitions in `packages/loot-core/src/types/`
+4. Use `satisfies` instead of `as` for type narrowing
+
+### Issue: Cannot find module or type definitions
+
+**Solution:**
+
+1. Check `tsconfig.json` for path mappings
+2. Verify platform-specific imports (`.web`, `.electron`, `.api`)
+3. Ensure you're using the correct import path for the package
+4. Run `yarn install` to ensure all dependencies are installed
+
+## Linter Errors
+
+### Issue: ESLint or Prettier errors
+
+**Solution:**
+
+1. Run `yarn lint:fix` to auto-fix many issues
+2. Check ESLint output for specific rule violations
+3. Common custom rules:
+   - `actual/no-untranslated-strings` - Add i18n translation
+   - `actual/prefer-trans-over-t` - Use Trans component instead of t() function
+   - `actual/prefer-logger-over-console` - Use logger instead of console
+4. Check `eslint.config.mjs` for complete rules list
+
+### Issue: Import order errors
+
+**Solution:**
+
+ESLint automatically organizes imports. The order should be:
+
+1. React imports (first)
+2. Built-in Node.js modules
+3. External packages
+4. Actual packages
+5. Parent imports
+6. Sibling imports
+7. Index imports
+
+Run `yarn lint:fix` to automatically fix import order.
+
+## Test Failures
+
+### Issue: Tests fail unexpectedly
+
+**Solution:**
+
+1. Check if test is running in correct environment (node vs web)
+2. For Vitest: check `vitest.config.ts` or `vitest.web.config.ts`
+3. For Playwright: check `playwright.config.ts`
+4. Ensure mock minimization - prefer real implementations over mocks
+5. **Lage cache issues**: Clear cache with `rm -rf .lage` if tests behave unexpectedly
+6. **Tests continue on error**: With `--continue` flag, all packages run even if one fails
+
+### Issue: Lage cache causing test issues
+
+**Solution:**
+
+```bash
+# Clear Lage cache
+rm -rf .lage
+
+# Run tests without cache
+yarn test:debug
+```
+
+### Issue: E2E tests fail
+
+**Solution:**
+
+1. Ensure Playwright browsers are installed: `yarn workspace @actual-app/web run playwright install`
+2. Run tests with headed browser for debugging: `yarn workspace @actual-app/web run playwright test --headed --debug`
+3. Check `playwright.config.ts` for configuration issues
+4. Verify test environment setup
+
+## Import Resolution Issues
+
+### Issue: Cannot resolve imports
+
+**Solution:**
+
+1. Check `tsconfig.json` for path mappings
+2. Check package.json `exports` field (especially for loot-core)
+3. Verify platform-specific imports (`.web`, `.electron`, `.api`)
+4. Use absolute imports in `desktop-client` (enforced by ESLint)
+5. Ensure you're importing from the correct package path
```

Comment:
IMO this is kind of the only step that is valid here, it's unlikely to be a tsconfig/exports issue